### PR TITLE
PSR-2 compliance

### PR DESCRIPTION
**Pull request by MAXakaWIZARD**
This PR introduces some efforts on making code PSR2-compliant. 
It would be great to have this in `2.2.0`

### CODE REVIEW

**Comment by Naktibalda** [comment]
Could you write a summary of what you did there?
Have you only changed the formatting?

**Comment by MAXakaWIZARD** [comment]
@Naktibalda Yes, those are first steps: formatting, splitting some long lines etc. I suppose nothing critical. I've talked to @DavertMik and he said he won't mind those changes. Next steps would be renaming methods and properties starting with underscore, splitting files with multiple classes etc. But I think this is different story, because those changes are likely to introduce BC issues (target for `3.0.0`?).

**Comment by MAXakaWIZARD** [comment]
@Naktibalda BTW, why `composer.lock` is not in `.gitignore` (both `2.1` and `master` branches)?

**Comment by Naktibalda** [comment]
It is a good question,
The file itself used to be in git, but I removed it just few months ago.

You are welcome to add composer.lock to .gitignore

**Comment by Naktibalda** [comment]
I reviewed the changes, thanks for cleaning up the mess that nobody wanted to clean.
My only concern is that it will be a lot more tedious to merge changes from 2.1 to master and we will probably stop supporting 2.1 very quickly.

Please rebase your branch with master.

**Comment by MAXakaWIZARD** [comment]
@Naktibalda Done. 
P. S. Why is this facebook test failing from time to time?

File: src/Codeception/PHPUnit/Runner.php [hunk 1]
```diff
@@ -125,19 +125,28 @@ protected function applyReporters(\PHPUnit_Framework_TestResult $result, array $
...
             codecept_debug('Printing JUNIT report into ' . $arguments['xml']);
-            self::$persistentListeners[] = $this->instantiateReporter('xml', [$this->absolutePath($arguments['xml']), false]);
+            self::$persistentListeners[] = $this->instantiateReporter(
+                'xml',
+                [$this->absolutePath($arguments['xml']), false]
+            );
         }
         if ($arguments['tap']) {
             codecept_debug('Printing TAP report into ' . $arguments['tap']);
             self::$persistentListeners[] = $this->instantiateReporter('tap', [$this->absolutePath($arguments['tap'])]);
```
Comment (Naktibalda) [inline src/Codeception/PHPUnit/Runner.php]: This line is inconsistent with other formats

File: src/Codeception/Step/Executor.php [hunk 2]
```diff
@@ -11,7 +11,8 @@ class Executor extends CodeceptionStep
 
     public function __construct(\Closure $callable, $arguments = [])
     {
-        // TODO: add serialization to function http://www.htmlist.com/development/extending-php-5-3-closures-with-serialization-and-reflection/
+        // TODO: add serialization to function
+        // http://www.htmlist.com/development/extending-php-5-3-closures-with-serialization-and-reflection/
```
Comment (Naktibalda) [inline src/Codeception/Step/Executor.php]: This TODO is 3 years old.

File: src/Codeception/PHPUnit/Runner.php (hunk 1 above)
Comment (MAXakaWIZARD) [inline src/Codeception/PHPUnit/Runner.php]: Sorry, I didn't get what you mean.

File: src/Codeception/Step/Executor.php (hunk 2 above)
Comment (MAXakaWIZARD) [inline src/Codeception/Step/Executor.php]: Ok, I've removed it

**Comment by DavertMik** [comment]
> Next steps would be renaming methods and properties starting with underscore, splitting files with multiple classes etc. 

He-he.. Nope. This won't happen :smiling_imp: At least in nearest future )

> Why is this facebook test failing from time to time?

Long story ) It just happens

I'd ask you to add a command to robofile which will run phpcs with your settings so everyone could run cs-fixer regularly. I will add it to pre-commit hook :)

**Comment by MAXakaWIZARD** [comment]
@DavertMik I can do that later, if you don't mind. Can we merge this PR because it will cause a lot of conflicts?
> I'd ask you to add a command to robofile which will run phpcs with your settings so everyone could run cs-fixer regularly. I will add it to pre-commit hook :)

**Comment by DavertMik** [comment]
As @Naktibalda mentioned we may have conflicts with 2.1 once it gets merged to master.
Can you do this PR for 2.1?

**Comment by MAXakaWIZARD** [comment]
@DavertMik You want me to discard this PR and redo it for `2.1`?

**Comment by DavertMik** [comment]
Yes. I will merge PR for 2.1 and then I merge 2.1 to master then we redo this PSR.
This is how it will work )

**Pull request closed without merging**